shell.nix

```diff
@@ -0,0 +1,52 @@
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
Impressive work here in such short time! Nix language is not the easiest to grasp